releng/buildpacks/auditlog-client/src/main/java/com/sap/auditlog/client/deserializers/InstantDeserializer.java

```diff
@@ -0,0 +1,15 @@
+package com.sap.auditlog.client.deserializers;
```

Comment:
Why not using Gson instead?

Reply:
hmm i dont think adding another dependency is justified for a single simple deserializer

Reply:
Also gson also requires specific deserializer for the Instant class, so it doesn't matter if Jackson or gson is used.

Reply:
Gson as of now is the default JSON parser/serializer in XSK. We can replace it with Jackson, but if possible let use only one.